uber-pack/README.md

```diff
@@ -0,0 +1,171 @@
+# Uber Package with AEM Connector
```

Comment:
Can we also update the main README. It should provide more information about the connector and provide instructions on how to setup the connectors locally or on prem.